cardano-node/src/Cardano/CLI/Delegation.hs

```diff
@@ -14,6 +14,9 @@ module Cardano.CLI.Delegation
 where
 
 import           Cardano.Prelude hiding (option, show, trace)
+
+import           Control.Monad.Trans.Except.Extra (left)
```

Comment:
`import           Control.Monad.Trans.Except.Extra (left)` should be moved as follows:
```
import           Control.Monad.Trans.Except.Extra (left)
import qualified Data.ByteString.Lazy as LB
```